### ISSUE TITLE
possible to detect missing aria-live or transitions?

### ISSUE THREAD

**Issue by rramsey** [dequelabs/axe-core-npm#1172]
### Product

cli

### Question

Hi,
I'm trying to learn playwright and axe-core for playwright. I have a sample vue application that basically has a paragraph with "hello, world" in it and a button that will translate it when you press a button. The html is something like:
````
<p v-if="lang==='english'">
  Hello, World
</p>
<p v-else>
  Hallo, Verden
</p>
<button
  @click="lang='norsk'
>
  Translate
</button>
````
Maybe I'm wrong, but it seems like there should be an `aria-live='polite'` on those paragraphs so that a screen reader will read the new text.

There's also transition on the paragraph. I tried to make it annoying, so there's a rotateY(180deg), translateX(20px), opacity:0, and background-color: tan, so it flips like a card, slides right, fades out, and changes background color. It's pretty horrible.

I create a new AxeBuilder and run it on the page but it doesn't flag either of those as issues, even when I am set to prefer-reduced-motion. 

Maybe I'm wrong about what's needed for wcag2.2aaa, but I would think that the screen reader should announce the new text and flag the transition as excessive.

I didn't see a way to have Axe start monitoring the page then have playwright click the button and wait for the animation to finish.

I don't think this is a bug, but it might be a feature request. Seems like this would be a good thing to check, but at the same time really difficult. 

Any  tips?

**Comment by straker** [comment]
Thanks for the issue. I don't think this is something we'll be able to do. 

First, to understand if there's a transition on the page would require us to parse all the contents of every CSS file and try to match all rules to elements on the page (since there's no easy way to tell which rules are currently being applied to an element). Even if we were to do that, there's an even bigger problem of knowing if there's an `@media` override that would disable the transition, or reduce it in such a way that it would not be a problem.

As for the `aria-live` part, that's also not something we could know as it would require understanding that the transition hides one element and then shows another. It also would require us to understand that the new text should be announced to screen readers, which depends on the context of the text and its purpose. Both of these things axe-core can't understand and requires a human to make the decision.

**Comment by rramsey** [comment]
No worries. I thought this seemed like a pretty hard thing to do. I figured a best case scenario would be to output a warning that something needs a human to check it out.
